app/src/main/java/com/example/solidarity/fragments/EventsFragment.java

```diff
@@ -0,0 +1,68 @@
+package com.example.solidarity.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.solidarity.Event;
+import com.example.solidarity.R;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+
+import java.util.List;
+
+
+public class EventsFragment extends Fragment {
+
+    public static final String TAG = "EventsFragment";
+    private RecyclerView rvEvents;
+
+
+    public EventsFragment() {
+        // Required empty public constructor
+    }
+
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_events, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        rvEvents = view.findViewById(R.id.rvEvents);
+
+        queryEvents();
+    }
+
+    private void queryEvents() {
+        ParseQuery<Event> query = ParseQuery.getQuery(Event.class);
+        query.include(Event.getKeyAuthor());
```

Comment:
getKeyAuthor tells me that this is a key of the json that is returned in the response. Name it better.